NSS-Day-Cohort-42/rare-media-moguls
# Conditional rendering implemented on Navbar.

#### Changes Made
1. Modified files `Nav.js` & `UserProvider.js` to conditionally render Navbar based on the logged in user.
1. Removed unused files per wireframe
1. Modified `Nav.css` per wireframe.

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-refactor-nav
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
4. Test app functionality.
> When logged in as an **Admin**, Navbar should render the following buttons:
![Screen Shot 2020-11-17 at 4 14 59 PM](https://user-images.githubusercontent.com/64630466/99457106-0e2ca100-28f0-11eb-8656-6ef67ff1fec6.png)
> When logged in as an **Author**, Navbar should render the following buttons:
![Screen Shot 2020-11-17 at 4 14 30 PM](https://user-images.githubusercontent.com/64630466/99457062-fe14c180-28ef-11eb-8db3-9b439cc88fe8.png)

5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
